lib/fresh_connection/version.rb

```diff
@@ -1,4 +1,4 @@
 module FreshConnection
-  VERSION = "2.3.0"
+  VERSION = "2.3.1"
```

Comment:
Actually, I do -- this came from the version # in my fork, which I need to bump in order to force bundle install to pull the updated code from github in our application that is using fresh_connection.